Power BI on raw CM exports spec (#5)
microsoft/cloud-hubs: docs/deploy/README.md

The FinOps toolkit includes multiple ARM templates depending on your needs. We recommend using the **FinOps hub with exports** template to streamline your setup process. All the examples below refer to this template. You may select a different template, which will change the available options.

For more details about toolkit options, refer to the [available templates](../templates).

> ### ⚠️ Important
>
> _The FinOps toolkit will change over time. We highly recommend saving a parameter file to ensure you can re-deploy new versions of the template with the same parameters._

On this page:

- [Using custom deployment in the Azure portal](#using-custom-deployment-in-the-azure-portal)
- [Deploy from Microsoft Learn code samples](#deploy-from-microsoft-learn-code-samples)
- [Deploy from Azure Quickstart Templates](#deploy-from-azure-quickstart-templates)
- [Future considerations](#future-considerations)

---

## Using custom deployment in the Azure portal

The Azure portal includes a **Custom deployment** option that supports all templates available in the Azure Quickstart Templates repository. To deploy a quickstart template:

> 1. Open [Custom deployment](https://portal.azure.com/#create/Microsoft.Template)

Reviewer: Not sure if it's important but the option in portal is "Deploy a custom template" than on the actual page as you point out it's "Custom deployment" might confuse some users the difference.

Author: I can share the feedback with the portal team. I'm just documenting what they have right now. I plan to add a separate proposal for what we should do in CM where we can do more.

      **subscription** | String | ID of the subscription to deploy the hub instance to.
      **hubName** | String | Name of the resource group and name prefix for all resources. Default: `finops-hub`.
      **location** | String | Azure location where all resources should be created.
      **exportScopes** | Array | Optional. List of scope IDs to create exports for.
5. Select the **Edit parameters** link at the top of the form.
6. Select the **Download** command at the top of the page to save your parameters file. Keep this for your next deployment.
7. Select the **Save** button at the bottom of the page.
8. Select the **Review + create** button.
9. Select the **Create** button.

If you received any validation errors, fix those and attempt to create the resources again.

<br>

## Deploy from Microsoft Learn code samples

Microsoft Learn hosts all templates available from the Azure Quickstart Templates repository. To deploy from Microsoft Learn:

1. Open the [FinOps hub code sample](https://learn.microsoft.com/en-us/samples/azure/azure-quickstart-templates/finops-hub-with-exports).
2. Select the **Deploy to Azure** button towards the top of the page.

> 3. Complete [steps 5 and beyond](#edit-params) from the custom deployment steps above.

Author: ```suggestion
3. Complete [steps 4 and beyond](#edit-params) from the custom deployment steps above.
```

<br>

## Deploy from Azure Quickstart Templates

Azure Quickstart Templates is a Microsoft-managed community of open source deployment templates. Many teams at Microsoft and across the community publish templates to share with the customers and partners.

Templates are organized based on scope:

- Tenant templates in [tenant-deployments](https://github.com/Azure/azure-quickstart-templates/tree/master/tenant-deployments)
- Resource group templates in [quickstarts/microsoft.costmanagement](https://github.com/Azure/azure-quickstart-templates/tree/master/quickstarts/microsoft.costmanagement)

We do not currently have any subscription or management group templates.

To deploy a quickstart template:

> 1. Open the [FinOps hub template folder](https://github.com/Azure/azure-quickstart-templates/tree/master/quickstarts/microsoft.costmanagement/finops-hub-with-exports).

Author: ```suggestion
1. Open the [FinOps hub with exports template](https://github.com/Azure/azure-quickstart-templates/tree/master/template-deployments/finops-hub-with-exports).
```

2. Select the **Deploy to Azure** button towards the top of the page.

> 3. Complete [steps 5 and beyond](#edit-params) from the custom deployment steps above.

Author: ```suggestion
3. Complete [steps 4 and beyond](#edit-params) from the custom deployment steps above.
```
